tests/testthat/test-logisticregutils.R

```diff
@@ -0,0 +1,34 @@
+context("Logistic Regression: helper functions testing ")
+
+config <- list(
+  `graph.resolution` = dropdownInput('%Question.graph.resolution%' , '1x'),
+  `the.link` = dropdownInput('%Question.Link%' , 'logit'),
+  `model.name` = textInput('%Question.Model Name%', 'my_model'),
+  `used.weights` = checkboxInput('%Question.Use Weights%' , FALSE),
+  `Weight Vec` = dropdownInput('%Question.Weight Vec%'),
+  `X Vars` = listInput('%Question.X Vars%', c('Sepal.Length', 'Petal.Length')),
+  `Y Var` = dropdownInput('%Question.Y Var%', 'Species')
+)
+
+if (config$the.link == "complementary log-log"){
+  config$the.link <- "cloglog"
+}
+
```

Comment:
The more I think about it, this line of code needs to be put below the `inputs` section. I will do it for the R code in the macro. The rationale here is that we want a clear separation of where the alteryx inputs/config ends and the R tool's logic starts.